server/modules/analysis/service/_rChain/rFunctions/index.js

```diff
@@ -0,0 +1,7 @@
+export { NA, dfVar, merge, setVar, sysTime } from './rFunctionsUtility'
+
+export { dbDisconnect, dbGetQuery, dbSendQuery, dbWriteTable, setConnection } from './rFunctionsDb'
+
+export { arenaStartTime, arenaEndTime, arenaInfo, arenaPersistCalculationScript } from './rFunctionsArena'
+
+export { arenaGet, arenaPost } from './rFunctionsArenaApi'
```

Comment:
omit the prefix rFunctions from the files

Reply:
ok